plugins/thumbnails/server/worker.py

```diff
@@ -157,3 +159,20 @@ def attachThumbnail(file, thumbnail, attachToType, attachToId, width, height):
     }
 
     return ModelImporter.model('file').save(thumbnail)
+
+
+def getImage(extension, data):
+    """
+    Check extension of image and opens it.
+
+    :param extension: The extension of the image that needs to be opened.
+    :param data: The image file stream.
+    """
+    if 'dcm' in extension:
+        # Open the dicom image
+        dicomData = dicom.read_file(six.BytesIO(data))
+        imageData = dicomData.pixel_array
+        return Image.fromarray(imageData).convert("I")
```

Comment:
Looks like this assumes a particular pixel format for the image data. Probably won't work for all DICOM images.